src/store/modules/errors.js

```diff
@@ -0,0 +1,17 @@
+const defaulState = {
+  message: '',
+  showError: false,
+}
+
+export default {
+  namespaced: true,
+  state: defaulState,
+  mutations: {
+    setMessage(state, message) {
+      state.message = message
+    },
+    setShowError(state, show) {
+      state.showError = show
+    },
+  },
```

Comment:
To simplify error handling and proper commit to the store

```suggestion
  },
  actions: {
    handleError({ commit }, { message }) {
      const error = new CustomError(message)
      commit('setError', error)
      commit('setShowError', true)
    },
}
```